pandas/tests/io/parser/test_python_parser_only.py

```diff
@@ -314,3 +315,26 @@ def test_malformed_skipfooter(python_parser_only):
     msg = "Expected 3 fields in line 4, saw 5"
     with pytest.raises(ParserError, match=msg):
         parser.read_csv(StringIO(data), header=1, comment="#", skipfooter=1)
+
+
+def test_comment_char_in_default_value(python_parser_only):
+    # GH#34002
+    from io import StringIO
+
+    data = (
+        "# this is a comment\n"
+        "1,2,3,4\n"
+        "1,2,3,4#inline comment\n"
+        "1,2#,3,4\n"
+        "1,2,#N/A,4\n"
```

Comment:
Would it be better to have the first row (not the commented one) to be like ``col_1,col_2,col_3,col4\n" to avoid confusion with the values below it?

Reply:
Good point, done